src/scales/scale.time.js

```diff
@@ -221,7 +221,10 @@ export default class TimeScale extends Scale {
 
   init(scaleOpts, opts) {
     const time = scaleOpts.time || (scaleOpts.time = {});
-    const adapter = this._adapter = new adapters._date(scaleOpts.adapters.date);
+    const adapterChartOpts = {
+      locale: opts.locale
+    };
+    const adapter = this._adapter = new adapters._date(scaleOpts.adapters.date, adapterChartOpts);
```

Comment:
I was thinking it could be better to pass only the locale and not an object with the locale. 
What are the other possible "chart" options which can affect a date time?